commit c54c17edee4f1156488ebdbd6ea31a3ef2eef44b (refs/changes/04/3104/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-12-12T15:56:42-08:00 (1 year, 10 months ago)
    
    TRITON-27 vm-agent: update eslint to 2.x (fix "make check" to update eslint if necessary)

diff --git a/Makefile b/Makefile
index 993edee..9eafbc2 100644
--- a/Makefile
+++ b/Makefile
@@ -142,7 +142,7 @@ dumpvar:
 
 # eslint ftw
 ESLINT = ./node_modules/.bin/eslint
-$(ESLINT): | $(NPM_EXEC)
+$(ESLINT): package.json | $(NPM_EXEC)
 	$(RUN_NPM_INSTALL)
 
 check:: check-eslint
